imap_processing/tests/spice/test_geometry.py

```diff
@@ -222,6 +245,45 @@ def test_frame_transform_az_el(mock_get_rotation_matrix, az_range, el_range):
     )
 
 
+@pytest.mark.parametrize(
+    "spice_frame",
+    [
+        SpiceFrame.IMAP_DPS,
+        SpiceFrame.IMAP_SPACECRAFT,
+        SpiceFrame.ECLIPJ2000,
+    ],
+)
+@pytest.mark.parametrize("degrees_bool", [True, False])
+def test_frame_transform_az_el_same_frame(spice_frame, degrees_bool):
```

Comment:
Since this calls out to the other method, I think this is testing the same thing as your other one so isn't needed? Your other test is great!

(I think I didn't quite realize what `degrees_bool` did before in my earlier comment and it doesn't actually apply a deg2rad transform between input/output, it just assumes both input/output are one or the other. i.e. I thought it would accept degrees input always and then transform to radians on the output.)

Reply:
I removed this test. Thanks!